### PR TITLE
Move vgpu nonspecific recipes to generic area

### PR DESCRIPTION
**Pull request by iartemenko**
Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

**Comment by andr2000** [comment]
you move files and add a new one...
IMO, these are two separate changes

**Comment by iartemenko** [comment]
Done

**Comment by andr2000** [comment]
Just for my understanding, why do we need to call cmake by hand here?

File: recipes-domx/meta-xt-images-domx/recipes-core/properties-cpp/properties-cpp_0.0.1.bb [hunk 1]
```diff
@@ -4,30 +4,27 @@ HOMEPAGE = "https://launchpad.net/properties-cpp"
...
-do_install() {
-    # Install header files
-    install -d ${D}/${includedir}/core
-    install -m 644 ${S}/include/core/*.h ${D}/${includedir}/core/
+inherit pkgconfig cmake
 
-    # Install pkgconfig file
-    install -d ${D}${libdir}/pkgconfig
-    install -m 644 ${S}/${PN}.pc ${D}${libdir}/pkgconfig/
+do_compile_prepend () {
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-domx/recipes-core/properties-cpp/properties-cpp_0.0.1.bb]: This is not needed

Comment (andr2000) [reply recipes-domx/meta-xt-images-domx/recipes-core/properties-cpp/properties-cpp_0.0.1.bb]: I mean cmake stuff below

File: recipes-domx/meta-xt-images-domx/recipes-core/properties-cpp/properties-cpp_0.0.1.bb [hunk 2]
```diff
@@ -4,30 +4,27 @@ HOMEPAGE = "https://launchpad.net/properties-cpp"
...
+     if [ -d ${S}/build ]; then
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-domx/recipes-core/properties-cpp/properties-cpp_0.0.1.bb]: pls remove this

**Review by andr2000 — APPROVED** [review]

**Pull request merged**